src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/McpServer/McpServer-CSharp/Program.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using McpServer_CSharp.Tools;
```

Comment:
Do we need a namespace? We could remove this line and the namespace line from the tools file. In "small" apps like this, I don't think namespaces add much, and someone who wanted to start from this and expand it could always add them later if they really wanted them.